determined-ai/determined
# feat: image updates: add tf 2.7; security tf 2.4, 2.5, 2.6; fix PTL.

## Description

Env image bump corresponding to https://github.com/determined-ai/environments/pull/125

## Test Plan

Should be covered with CI tests. Manual checks:
- "Everything" should still work.
- Run your favorite keras model on TF 2.7 image.
- Run PTL models using metrics from torchmetrics e.g. `e2e_tests/tests/fixtures/pytorch_lightning_amp/`.

## Commentary (optional)


## Checklist

See [Release Note](../docs/release-notes/README.md) for details.
